CMakeLists.txt

```
@@ -1,4 +1,5 @@
 cmake_minimum_required(VERSION 3.14)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```
Conan v2 requires CMake 3.15 or greater anyway.